<launch>
 <include file="$(find person_detection)/launch/cam.launch"/>
 <include file="$(find darknet_ros)/launch/darknet_ros.launch"/>
 <node pkg="person_detection" type="filter_and_pub.py" name="human_boxes"/>
 
 
</launch>

